src/distribution/normal.rs

```diff
@@ -508,4 +535,17 @@ mod tests {
         test_almost(5.0, 2.0, 10.0, 1e-14, inverse_cdf(0.9937903346742238648330218954258077788721022530769078));
         test_case(5.0, 2.0, f64::INFINITY, inverse_cdf(1.0));
     }
+
+    #[test]
+    fn test_default() {
+        let n = Normal::default();
+
+        n_mean = n.mean().unwrap();
+        n_std  = n.std_dev().unwrap();
```

Comment:
These two lines need a `let`:

```suggestion
        let n_mean = n.mean().unwrap();
        let n_std  = n.std_dev().unwrap();
```

Reply:
Oops not sure how I missed this one (cargo test ran??). Fixed.

Reply:
If you scroll up, you'll see that the entire tests module is only run when the `nightly` feature flag is enabled. So you'd need to run `cargo +nightly test -F nightly`, which is easy to miss